apps/token/src/styles.css

```diff
@@ -1,3 +1,15 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+@layer base {
```

Comment:
@mattrussell36 I think this might be controversial with you, but having played with styled components to achieve the same result for these text elements, I think this is a far better solution for the following reasons:

1. Tailwind applies these classes directly as CSS as they form part of the base styling, with no classes coming through into the DOM. This means we can apply Tailwind classes to modify styling without collisions.
2. It significantly reduces clutter in the form of hundreds of classes added to the DOM to achieve the same effect.
3. It's simpler than custom styled components. Fresh devs don't need to understand to import a special component to use, instead relying on the h1, p tags etc they'd expect.

Reply:
chore (non blocking): As discussed on slack I'd prefer to add a wrapping class to apply styles to elements, this makes our config diverge less between apps.

```
.text-flow p {
  @apply mb-12
}
```